express-notes-api/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const dataJSON = require('./data.json');
+const fs = require('fs');
+
+const app = express();
+const middlewareJSON = express.json();
+
+app.get('/api/notes', (req, res) => {
+  const notesArray = [];
+  for (const property in dataJSON.notes) {
+    notesArray.push(dataJSON.notes[property]);
+  }
+  res.status(200).json(notesArray);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = req.params.id;
+  const note = dataJSON.notes[id];
+  if (id < 1) {
```

Comment:
you want to check specifically if id isNaN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

You also want to confirm the number is an integer value: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger